tests/fib/test_fib.py

```diff
@@ -25,6 +26,8 @@
     pytest.mark.topology('any')
 ]
 
+PTF_PORT_MAPPING_MODE = "use_sub_interface"
```

Comment:
this test is shared by other topologies and this setting might not affect them since the ptf nn agent is already started using the regular interface and we don't restart it in this test. But this is misleading.
We should just set it in modules using ptf runner where we need to use regular interfaces for backend topology and make sure we restart with subintf at the end of the module 

These are the cases I can think of using ptf runner
1. all topologies other than backend should use regular interface.
2. majority of tests using backend topology should use sub interface
3. few tests using backend topology should use regulare interface. So I think we should just do handling for this case since ptf is already started with the correct interface during deployment

Reply:
1. this setting in `test_fib` aims to change the behavior of `ptf_runner`.
2. since `ptfadapter` is a module-level fixture, setting the mode per-module is sufficient to change the behavior.

I think changing the default mode to `use_sub_interface` could make it less misleading because the majority of testcases won't have to define this global module-level variable.

Reply:
hi @neethajohn, I've separate the original PR into two PRs, one is for `ptf_runner`, the another is for `ptfadapter`.